### DISCUSSION TITLE
Separating "simple" and "bfb" tags

### DISCUSSION THREAD

**Discussion by samsrabin** [ESCOMP/CTSM#2162] [General]
It might be useful to separate the "[tag: simple bfb](https://github.com/ESCOMP/CTSM/labels/tag%3A%20simple%20bfb)" label into "tag: simple" and "tag: bfb". I thought about this because #2125 is bit-for-bit (as it only affects testing infrastructure) but it was _not_ simple. 

I don't think we'd have to retroactively go through existing issues/PRs with the tag to decide which they should get, but it could be useful going forward. We could even keep the existing label for issues/PRs that really are both simple and bit-for-bit.

What do y'all think?

**Comment by wwieder** [comment]
This seems fine if it helps clarify aspects of our workflow.  I agree, it's not worth revising existing issues or PRs, but good to apply moving forward.  I'd say go ahead and make this label change, Sam.  Thanks.

**Comment by ekluzek** [comment]
Hi @samsrabin actually coupling "simple" and "bfb" was an intentional part of this label. The idea is that "simpleb4b" tags are things easy for anyone to work on. If something is NOT b4b we treat it totally different than something that is "b4b". We are more likely to have answer changes in seperate tags. Whereas "simpleb4b" are things that bringing together makes sense. It can be "b4b" and NOT simple and that's likely to be something you would NOT merge together with other "simpleb4b" changes. So "simpeb4b" is a way to identify things that are easy to put together. So I do like it as a single thing.

**Reply by samsrabin** [reply]
Sure, that's why I suggested keeping the current label as an option. But as you said, there are some things that are just one or the other, and I think it'd be useful to have the separate labels as well.

**Reply by ekluzek** [reply]
Ok. I have wondered about a label for "answer changing". Since we treat all of those cases much different than anything else. I think I like it expressed that way over "not b4b".

So I suggest adding:

Answer changing
Simple but answer changing

And then they'll still be simplebfb so that covers the space, but makes it more clear on which should be used. And I suppose the second one above should be shorter....

**Comment by samsrabin** [comment]
I've gone ahead and added [tag: simple](https://github.com/ESCOMP/CTSM/labels/tag%3A%20simple) and [tag: bfb](https://github.com/ESCOMP/CTSM/issues?q=label%3A%22tag%3A+bfb%22) labels. All three labels are summarized [here](https://github.com/ESCOMP/CTSM/labels?q=simple+bfb).
